cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js

```diff
@@ -215,36 +220,52 @@ describe('Patron Block: Maximum outstanding fee/fine balance', () => {
         cy.getAdminToken();
       })
       .then(() => {
-        cy.get('@items').each((item) => {
-          Checkout.checkoutItemViaApi({
-            id: uuid(),
-            itemBarcode: item.barcode,
-            loanDate: moment.utc().format(),
-            servicePointId: testData.userServicePoint.id,
-            userBarcode: userData.barcode,
-          });
-        });
+        cy.login(userData.username, userData.password);
+        cy.visit(SettingsMenu.conditionsPath);
+        Conditions.waitLoading();
```

Comment:
please check if it is possible here to substitute three lines in one line

Reply:
unfortunately it won't work. When you go directly to this section, it is not displayed properly